config.ru

```diff
@@ -1,4 +1,12 @@
 require './cheeks_in_seats'
+require "rubygems"
```

Comment:
Everything From Line 2 to 9 needs to go in cheeks_in_seats.rb

Reply:
Please see PR just put in on this branch/PR with updates requested. Thanks